mijia/src/bluetooth.rs

```diff
@@ -109,19 +109,38 @@ impl BluetoothSession {
         ))
     }
 
-    /// Power on the bluetooth adapter adapter and start scanning for devices.
+    /// Power on all bluetooth adapters and start scanning for devices.
     pub async fn start_discovery(&self) -> Result<(), eyre::Error> {
-        let adapter = dbus::nonblock::Proxy::new(
+        let bluez_root = dbus::nonblock::Proxy::new(
             "org.bluez",
-            "/org/bluez/hci0",
+            "/",
```

Comment:
We might be able to  start at "/org/bluez" but I haven't checked.

Reply:
That appears not to work.